recipes/recipes_emscripten/nelson/recipe.yaml

```diff
@@ -1,19 +1,20 @@
 context:
-  version: 0.0.1
+  version: 0.3.0
 
 package:
   name: nelson
   version: '{{ version }}'
 
 source:
-  git_url: https://github.com/Nelson-numerical-software/nelson-minimalist-core.git
-  patches:
-    - patches/01-boyer_search.patch
-    - patches/02-no_fcloseall.patch
-    - patches/03-dynamic_error.patch
+  url: https://github.com/Nelson-numerical-software/nelson-minimalist-core/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: d9212109b54c677423ff1e3998c30d3dd9f165734b8fc40cc279dce961e0edd2
+  #patches:
+    #- patches/01-boyer_search.patch
+    #- patches/02-no_fcloseall.patch
+    #- patches/03-dynamic_error.patch
```

Comment:
We could maybe have removed them?

Reply:
Yes I was rushing to have something to show, I'll open a PR to remove them